modules/ROOT/pages/dw-core-types.adoc

```diff
@@ -130,7 +130,7 @@ So when it is mapped to DataWeave values, it is wrapped in a Null with a Schema
 
 | Null
 | `type Null = Null`
-| A Null type.
+| A Null type, which is for `null` values.
```

Comment:
```suggestion
| A Null type, which represents the `null` value.
```
It represents just one value that is null (but that value can be used in multiple places). Just a detail.